examples/dynamic_api.rs

```diff
@@ -52,18 +52,22 @@ async fn print_group(
         if !ar.verbs.contains(&"list".to_string()) {
             continue;
         }
-        let mut resource = DynamicResource::from_api_resource(&ar, &apis.group_version);
-        if ar.namespaced {
+        let gvk = GroupVersionKind::from_api_resource(&ar, &apis.group_version);
+        let api: Api<DynamicObject> = if ar.namespaced {
             if let Some(ns) = ns_filter {
-                resource = resource.within(ns);
+                Api::namespaced_with(client.clone(), ns, gvk)
+            } else {
+                Api::all_with(client.clone(), gvk)
```

Comment:
This is slightly awkward way of conditionally setting the namespace. Am considering an `Api::namespace(mut self, ns)` setter, but that would either break existing interface, or be one levenstein distance from `Api::namespaced`